pkg/controller/core/localqueue_controller.go

```diff
@@ -36,17 +36,17 @@ import (
 	"sigs.k8s.io/kueue/pkg/queue"
 )
 
-// QueueReconciler reconciles a Queue object
-type QueueReconciler struct {
+// LocalQueueReconciler reconciles a Queue object
```

Comment:
I thought I commented on this

```suggestion
// LocalQueueReconciler reconciles a LocalQueue object
```

Reply:
and lots more in this file related to logs